base_layer/core/src/blockheader.rs

```diff
@@ -62,6 +63,20 @@ impl BlockHeader {
     pub fn validate_pow(&self) -> bool {
         unimplemented!();
     }
+
+    pub fn create_empty() -> BlockHeader {
```

Comment:
it's more idiomatic for a fn like this to to be
`impl Default for BlockHeader...`